roles/foreman/defaults/main.yml

```diff
@@ -1,11 +1,2 @@
----
-
 foreman_provisioning_password: changeme
 foreman_admin_password: changeme
-authorized_keys: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
```

Comment:
I would leave this variables in, so that the role has a complete definition of the variables it consumes. It's kind of a pain point of ansible but I prefer to have redundant information than roles that don't define what they need.